### ISSUE TITLE
Quick install via docker not working

### ISSUE THREAD

**Issue by mikebakke** [wekan/wekan#2916]
## Issue
I have used this 2 line docker install to try and have a quick look at Wekan but cannot get the system up
docker run -d --restart=always --name wekan-db mongo:4.0

docker run -d --restart=always --name wekan --link "wekan-db:db" -e "WITH_API=true" -e "MONGO_URL=mongodb://my-ip:27017" -e "ROOT_URL=http://my-ip:8080" -p 8080:8080 quay.io/wekan/wekan

**Server Setup Information**: AMD 64 VM on Linux Mint

* Did you test in newest Wekan?: Have tried :latest

* Operating System: Linux Mint x64
* Deployment Method(snap/docker/sandstorm/mongodb bundle/source): docker
* Http frontend if any (Caddy, Nginx, Apache, see config examples from Wekan GitHub wiki first): none
* MongoDB Version: 4.0
* Wekan only works on newest desktop Firefox/Chromium/Chrome/Edge/Chromium Edge and mobile Chrome. What webbrowser version are you using? Latest firefox

**Problem description**:
Mongo starts and says listening for connections on 27017, Wekan fails to start. If I do a netstat -tulpn there is no open port 27017 so I assume this is why Wekan will not connect

- *If using Docker, what does show command `sudo docker logs wekan-db` ? Please anonymize logs.*
2020-02-04T18:40:05.047+0000 I CONTROL  [initandlisten] MongoDB starting : pid=1 port=27017 dbpath=/data/db 64-bit host=00c2037bdef3
2020-02-04T18:40:05.048+0000 I CONTROL  [initandlisten] db version v4.0.16
2020-02-04T18:40:05.048+0000 I CONTROL  [initandlisten] git version: 2a5433168a53044cb6b4fa8083e4cfd7ba142221
2020-02-04T18:40:05.048+0000 I CONTROL  [initandlisten] OpenSSL version: OpenSSL 1.0.2g  1 Mar 2016
2020-02-04T18:40:05.048+0000 I CONTROL  [initandlisten] allocator: tcmalloc
2020-02-04T18:40:05.048+0000 I CONTROL  [initandlisten] modules: none
2020-02-04T18:40:05.048+0000 I CONTROL  [initandlisten] build environment:
2020-02-04T18:40:05.048+0000 I CONTROL  [initandlisten]     distmod: ubuntu1604
2020-02-04T18:40:05.048+0000 I CONTROL  [initandlisten]     distarch: x86_64
2020-02-04T18:40:05.048+0000 I CONTROL  [initandlisten]     target_arch: x86_64
2020-02-04T18:40:05.048+0000 I CONTROL  [initandlisten] options: { net: { bindIpAll: true } }
2020-02-04T18:40:05.055+0000 I STORAGE  [initandlisten] Detected data files in /data/db created by the 'wiredTiger' storage engine, so setting the active storage engine to 'wiredTiger'.
2020-02-04T18:40:05.056+0000 I STORAGE  [initandlisten]
2020-02-04T18:40:05.056+0000 I STORAGE  [initandlisten] ** WARNING: Using the XFS filesystem is strongly recommended with the WiredTiger storage engine
2020-02-04T18:40:05.056+0000 I STORAGE  [initandlisten] **          See http://dochub.mongodb.org/core/prodnotes-filesystem
2020-02-04T18:40:05.056+0000 I STORAGE  [initandlisten] wiredtiger_open config: create,cache_size=256M,cache_overflow=(file_max=0M),session_max=20000,eviction=(threads_min=4,threads_max=4),config_base=false,statistics=(fast),log=(enabled=true,archive=true,path=journal,compressor=snappy),file_manager=(close_idle_time=100000),statistics_log=(wait=0),verbose=(recovery_progress),
2020-02-04T18:40:05.731+0000 I STORAGE  [initandlisten] WiredTiger message [1580841605:726892][1:0x7f16b274fa80], txn-recover: Main recovery loop: starting at 1/24064 to 2/256
2020-02-04T18:40:05.828+0000 I STORAGE  [initandlisten] WiredTiger message [1580841605:828081][1:0x7f16b274fa80], txn-recover: Recovering log 1 through 2
2020-02-04T18:40:05.886+0000 I STORAGE  [initandlisten] WiredTiger message [1580841605:886871][1:0x7f16b274fa80], txn-recover: Recovering log 2 through 2
2020-02-04T18:40:05.924+0000 I STORAGE  [initandlisten] WiredTiger message [1580841605:924178][1:0x7f16b274fa80], txn-recover: Set global recovery timestamp: 0
2020-02-04T18:40:06.067+0000 I RECOVERY [initandlisten] WiredTiger recoveryTimestamp. Ts: Timestamp(0, 0)
2020-02-04T18:40:06.118+0000 I CONTROL  [initandlisten]
2020-02-04T18:40:06.118+0000 I CONTROL  [initandlisten] ** WARNING: Access control is not enabled for the database.
2020-02-04T18:40:06.118+0000 I CONTROL  [initandlisten] **          Read and write access to data and configuration is unrestricted.
2020-02-04T18:40:06.119+0000 I CONTROL  [initandlisten]
2020-02-04T18:40:06.191+0000 I FTDC     [initandlisten] Initializing full-time diagnostic data capture with directory '/data/db/diagnostic.data'
2020-02-04T18:40:06.223+0000 I NETWORK  [initandlisten] waiting for connections on port 27017

NOTE: 27017 does not show in netstat at this point

While Mongo is running, I start Wekan and get this:
root@testbed-vm-i3:~# docker run  --name wekan --link "wekan-db:db" -e "WITH_API=true" -e "MONGO_URL=mongodb://localhost:27017/db" -e "ROOT_URL=http://192.168.0.52:8080" -p 8080:8080 quay.io/wekan/wekan
/build/programs/server/node_modules/fibers/future.js:313
                                                throw(ex);
                                                ^

MongoNetworkError: failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1134:16) {
  name: 'MongoNetworkError',
  errorLabels: [Array],
  [Symbol(mongoErrorContextSymbol)]: {}
}]
    at Pool.<anonymous> (/build/programs/server/npm/node_modules/meteor/npm-mongo/node_modules/mongodb-core/lib/topologies/server.js:431:11)
    at Pool.emit (events.js:223:5)
    at /build/programs/server/npm/node_modules/meteor/npm-mongo/node_modules/mongodb-core/lib/connection/pool.js:557:14
    at /build/programs/server/npm/node_modules/meteor/npm-mongo/node_modules/mongodb-core/lib/connection/connect.js:39:11
    at callback (/build/programs/server/npm/node_modules/meteor/npm-mongo/node_modules/mongodb-core/lib/connection/connect.js:261:5)
    at Socket.<anonymous> (/build/programs/server/npm/node_modules/meteor/npm-mongo/node_modules/mongodb-core/lib/connection/connect.js:286:7)
    at Object.onceWrapper (events.js:313:26)
    at Socket.emit (events.js:223:5)
    at emitErrorNT (internal/streams/destroy.js:92:8)
    at emitErrorAndCloseNT (internal/streams/destroy.js:60:3)
    at processTicksAndRejections (internal/process/task_queues.js:81:21) {
  name: 'MongoNetworkError',
  errorLabels: [ 'TransientTransactionError' ],
  [Symbol(mongoErrorContextSymbol)]: {}
}

I am a relative novice so apologies for stupid errors....

**Comment by xet7** [comment]
It seems that `--link` is legacy feature: https://docs.docker.com/network/links/
I could not get docker commands from https://github.com/wekan/wekan/wiki/Docker
working correctly. Maybe someone more familiar with Docker could help?

Can you use docker-compose.yml from https://github.com/wekan/wekan ?
Edit that file and then:
```
docker-compose up -d
```

**Comment by mikebakke** [comment]
Hi

I did try the compose option first, again from the Wekan wiki and that failed with a different error so I simplified to use those 2 commands also from the wiki. The example composer I found specified a much earlier release of MongoDB.

I guess my real confusion is that just looking at Mongo in isolation, it says it's listening on 27017 and a docker ps shows that port but netstat simply doesn't see it. Most strange.

I will have another crack at the original compose example but update Mongo to latest and see if that helps. 

Failing that I guess I give up and go back to Kanboard which would seem a shame.

Regards

Mike
